### ISSUE TITLE
Some HD mod more or less important fixes/features

### ISSUE THREAD

**Issue by dydzio0614** [vcmi/vcmi#1290]
Copypasted from another person who plays multiplayer

- [x] Battlefield casualties is not checkable option (it unimplemented?) I wanna see how many will be dead by attack and by spell, not just plain damage. (it's not HD, it's HotA, but it is super nice. it probably top 1 feature people want in SoD from HotA). (#857)
- [ ] hotkeys (army manipulation do work in hero main screen, but no mini screen at the left. also it's very clunky compared to HD, just do common things with ctrl click and alt click, you often get wrong results while in HD mod it works smoothly) + switching heroes via ctrl-click
- [x] move points (everywhere, path length, remain, mp on hero hover, showing track possible for next day (it's Hota not HD mod feature btw) - DONE IN BASIC FORM
- [x] showing stack sizes of guards (20-49 etc) - assuming no extending creature bank info for now
- [x] quick pickups (i don't want popup dialogs to exist to pickup artifact, scroll, get spell, etc etc)
- [x] hero exchange in town - done, but not ideal
- [ ] hero exchange in town in HD mod style
- [ ] artifacts bag in hero screen and in hero exchange screen
- [ ] spellbook (more spells, no mandatory stupid wrap page animation, show all spells damage on click)
- [x] animation control, instant movement is unavailable (max speed setting is TOO SLOW). that is for hero on adventure map (both player and AI) and battle movement too
- [ ] showing possible movement of creatures in the battle (sorry 50 shaded of grey is not enough). also nice to have 'move if unblocked' hex added recently this year in HD i believe.
- [x] usable battle order queue. it shows move order, but have 0 connection with creatures on the field (which is most important). so i need to hover on stack and see its order and vice versa. (#806)
also its VERY HUGE , i wanna it be smaller as in HD mod. and no ability to hide battle order it the settings is annoying too. 
- [ ] no battle log expansion by clicking on it (show everything in extra window) (#925)
- [x] showing grid on adventure map

**Comment by misiokles** [comment]
As far I remember - battle order can be hide anytime during battle by pressing Q.

**Comment by krs0** [comment]
> 

it still should be in a menu with associated hotkey described.

**Comment by dydzio0614** [comment]
grid is implemented via "set showGrid on" client command - friendlier UI is all that's needed for this

**Comment by kambala-decapitator** [comment]
regarding move points, I think it would also be useful to display movement cost to spells (Town Portal, Dimension Door)

**Comment by DjWarmonger** [comment]
We'll need larger spellbook graphics, like this:
![Large Spellbook](https://user-images.githubusercontent.com/2566990/212984306-383fdad6-e5ee-40cd-9b64-358487aff959.png)

Actually I didn't realize it's a big issue until someone pointed it out during stream.

It becomes more of a problem when more spells are added via mods.

**Comment by dydzio0614** [comment]
no mandatory page wrap for spells is done

**Comment by rilian-la-te** [comment]
@dydzio0614 quick pickups is merged.

**Comment by IvanSavenko** [comment]
Probably closing - moved to Github Project
https://github.com/orgs/vcmi/projects/2
